design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -81,6 +85,14 @@ input SetNegotiableQuoteShippingAddressInput {
     )
 }
 
+input SetNegotiableQuoteBillingAddressInput {
+    quote_uid: ID! @doc(description: "ID obtained from NegotiableQuote type")
+    customer_address_id: Int @doc(
+        description: "ID obtained from the CustomerAddress type. A new address can be added using Mutation.createCustomerAddress"
+    )
+    new_customer_address: CustomerAddressInput @doc(description: "New address to be added")
```

Comment:
`SetBillingAddressonCartInput` takes in the CartAddressInput for billing_address. Do we need all the additional fields provided by `CustomerAddressInput` for negotiableQuote, new_customer_address?
@pdohogne-magento  @amolina-magento : what do you guys think?

Reply:
I've changed these inputs to match Cart, which also address the `same_as_shipping` comment below

Reply:
The `SetNegotiableQuoteBillingAddressInput` has been updated to reuse the `BillingAddressInput` from QuoteGraphQL. Given our recent conversations, this looks good for now.